modules/account-lib/src/coin/algo/utils.ts

```diff
@@ -0,0 +1,82 @@
+import algosdk from 'algosdk';
+import tweetnacl from 'tweetnacl';
+import * as hex from '@stablelib/hex';
+
+/**
+ * Returns whether or not the string is a valid protocol address
+ *
+ * @param {string} address - the address to be validated
+ * @returns {boolean} - the validation result
+ */
+export function isValidAddress(address: string): boolean {
+  return algosdk.isValidAddress(address);
+}
+
+/**
+ * Returns whether or not the string is a valid protocol transaction id or not.
+ *
+ * @param {string} txId - the transaction id to be validated
+ * @returns {boolean} - the validation result
+ */
+export function isValidTransactionId(txId: string): boolean {
+  if (txId.length !== 104) return false;
```

Comment:
How did we come up with 104? Isn't it 52 characters?

https://algoexplorer.io/tx/CLICZYBANKD3IYA3YTGSVAHGCR2O6BHN3ZRU6VXOHUAJL5JTDYNA